src/view/conditions/components/__tests__/multipleItemEditor.test.jsx

```diff
@@ -13,13 +13,16 @@
 import React from 'react';
```

Comment:
Should you move the test to match the location of the subject? `src/view/components/__tests__/multipleItemEditor.test.jsx`

Reply:
Yes, thank you.